quadrat/theme.json

```diff
@@ -10,6 +10,24 @@
 			"area": "footer"
 		}
 	],
+	"customTemplates": [
+		{
+			"name": "blank",
+			"title": "Blank",
+			"postTypes": [
+				"page",
+				"post"
+			]
+		},
+		{
+			"name": "header-footer-only",
+			"title": "Header and Footer Only",
+			"postTypes": [
+				"page",
+				"post"
+			]
+		}
+	],
```

Comment:
I know these come from a previous PR but do we want these on the children themes?

Reply:
I wondered that too. What do you think @kjellr?